src/app/components/transferDetailView/index.tsx

```diff
@@ -64,23 +117,71 @@ function TransferDetailView({
   address,
   hideAddress,
   hideCopyButton,
+  outputScript,
+  outputScriptIndex,
 }: Props) {
+  const { t } = useTranslation('translation', { keyPrefix: 'CONFIRM_TRANSACTION' });
+  const [showScriptOutput, setShowScriptOutput] = useState(false);
+
+  const onButtonClick = () => {
+    setShowScriptOutput(true);
+  };
+
+  const onCrossClick = () => {
+    setShowScriptOutput(false);
+  };
+
   return (
-    <RowContainer>
-      {icon && <Icon src={icon} />}
-      {amount ? (
-        <ColumnContainer>
-          <AmountText>{amount}</AmountText>
-          {children}
-        </ColumnContainer>
-      ) : (
-        <TitleText>{title}</TitleText>
+    <>
+      {showScriptOutput && (
+        <ShowScriptBackgroundContainer>
+          <CrossContainer onClick={onCrossClick}>
+            <TransparentButton>
+              <img src={Cross} alt="cross" />
+            </TransparentButton>
+          </CrossContainer>
+          <ScriptOutputHeadingText>{`${t(
+            'SCRIPT_OUTPUT',
+          )} #${outputScriptIndex}`}</ScriptOutputHeadingText>
+          <ScriptText>script: btc.Script.encode {'{['}</ScriptText>
+          {outputScript &&
+            outputScript.map((script) => {
+              if (script instanceof Uint8Array) {
+                return (
+                  <ScriptText>
+                    {`new TextEncoder().encode (${new TextDecoder().decode(script)})`},
+                  </ScriptText>
+                );
+              } else {
+                return <ScriptText>{script},</ScriptText>;
+              }
+            })}
+
+          <ScriptText>{']}'}</ScriptText>
+        </ShowScriptBackgroundContainer>
```

Comment:
usually you would put modals at after the other elements, so the default visibility is already higher than the preceding elements, and it may mean you don't need to explicitly set the z-index: 10 on L92